tests/test-intermediate-ca.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+from vm import VM
+
+import os
+import pytest
+import time
+
+# Provide a list of VMs you want to reuse. VMs should have already microk8s installed.
+# the test will attempt a refresh to the channel requested for testing
+# reuse_vms = ['vm-ldzcjb', 'vm-nfpgea', 'vm-pkgbtw']
+reuse_vms = None
+
+# Channel we want to test. A full path to a local snap can be used for local builds
+channel_to_test = (
+    "latest/edge/intermediate-ca"  # os.environ.get("CHANNEL_TO_TEST", "latest/stable")
```

Comment:
This branch contains the fixes for the intermediate CA verification (https://github.com/canonical/microk8s/pull/4256).

Hard-coding this here is a temporary hack to see if the CI passes